dot/rpc/modules/dev.go

```diff
@@ -54,3 +60,23 @@ func (m *DevModule) Control(r *http.Request, req *[]string, res *string) error {
 	}
 	return err
 }
+
+// SetAuthorities dev rpc method that sets authorities for block producer
+func (m *DevModule) SetAuthorities(r *http.Request, req *[]interface{}, res *string) error {
```

Comment:
can you update this to `SetBABEAuthorities`?

Reply:
I can, but I didn't include BABE because this calling set authorities on the block producer interface SetAuthorities.  Maybe setBlockProducerAuthorities?  Or I can do SetBABEAuthorities as your asking, but I just wanted to comment.

Reply:
sure either one is fine, it's because we will probably need a similar function for updating grandpa authorities later